test/e2e/json-rpc/switchEthereumChain.spec.js

```diff
@@ -158,7 +158,36 @@ describe('Switch Ethereum Chain for two dapps', function () {
         });
 
         await driver.switchToWindowWithUrl(DAPP_ONE_URL);
+        // Switch to Dapp One and connect it
+        await driver.switchToWindowWithUrl(DAPP_URL);
```

Comment:
Nit: Can we remove the prior `await driver.switchToWindowWithUrl(DAPP_ONE_URL);` call? It seems that we're switching to the 2nd dapp for no reason prior to switching to the first here.

Reply:
Yes, Updated here https://github.com/MetaMask/metamask-extension/pull/27980/commits/84afa0627172d542d511ea9c480f9993c9417291